vignettes/analysing-n-grams.Rmd

```diff
@@ -207,7 +214,7 @@ flagship_journals <- without_wall %>%
 
 
 # Importing bigrams
-
+(#5)
```

Comment:
here I found a striking similarity between "bigram" and "bikram" (yoga), and the fact that I do not have any other thought connecting anything to bigram - you might want to introduce the term properly, at some point...(or supply a link to docs, wikipedia or whatever)
Additionally, the following paragraph is rather unclear to me, especially the last sentence. 
And, how do you know/distinguish the "relevant files on disk" anyways?